gymnasium/envs/registration.py

```diff
@@ -526,6 +526,7 @@ def make(
     autoreset: bool = False,
     apply_api_compatibility: Optional[bool] = None,
     disable_env_checker: Optional[bool] = None,
+    allow_default_wrappers: Optional[bool] = True,
```

Comment:
I personally dislike this, `make` already has enough parameters
If a `spec_stack` is passed to `make`, we should bypass the standard wrappers applied and only use the `WrapperSpec`, imo

Reply:
"If a spec_stack is passed to make, we should bypass the standard wrappers applied and only use the WrapperSpec, imo"

Yes, this is my motivation for `allow_default_parameters`, otherwise `gym.make(envspec)` won't bypass the standard wrappers. What alternative solution do you propose? I'm struggling to think of any.

Reply:
Should not a simple type check work? 
If string, look up Env spec
If Env spec, use Env spec
If sequence, extract wrappers specs and Env spec

Reply:
Env spec includes kwargs like max_steps etc which automatically construct the standard wrappers. Note that the entire spec_stack isn't passed to make(), only the EnvSpec element in the spec_stack.

Reply:
> Env spec includes kwargs like max_steps etc which automatically construct the standard wrappers. Note that the entire spec_stack isn't passed to make(), only the EnvSpec element in the spec_stack.

I don't understand? I thought the purpose of the spec stack was to have a list of wrapper / env specifications that can be saved and from the full environment stack could be recreated. Therefore, should we not provide capacity with `make` to pass the whole spec stack

Reply:
I'm with you. Originally I had a separate `recreate_env()` function but I've now updated `gym.make()` to work with `SpecStack`s